manage.py

```diff
@@ -5,4 +5,4 @@
 if __name__ == "__main__":
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    execute_from_command_line(sys.argv)
```

Comment:
We should revert this change

Reply:
should i removed this line?
`execute_from_command_line(sys.argv)`

Reply:
I've added new line in the end

Reply:
I've added new line